README.md

```diff
@@ -33,31 +33,54 @@ The Cloud Storage sink connector supports the following properties.
+| `timePartitionDuration`         | String  | False    | "86400000"   | The time interval for time-based partitioning, support formatted interval string, such as `30d`, `24h`, `30m`, `10s`, also support number in milliseconds precision, such as `86400000` refers to `24h` or `1d`. |
+| `batchSize`                     | int     | False    | 10           | The number of records submitted in batch.                                                                                                                                                                        |
+| `batchTimeMs`                   | long    | False    | 1000         | The interval for batch submission.                                                                                                                                                                               |
+| `sliceTopicPartitionPath`       | Boolean | False    | false        | When it is set to `true`, split the partitioned topic name into separate folders in the bucket path.                                                                                                             |
+| `withMetadata`                  | Boolean | False    | false        | Save message attributes to metadata.                                                                                                                                                                             |
+| `useHumanReadableMessageId`     | Boolean | False    | false        | Use human-readable format string for messageId in message metadata, the messageId will be in format like `ledgerId:entryId:partitionIndex:batchIndex`. Otherwise, the messageId will be in Hex-encoded string.   |
+| `withTopicPartitionNumber`      | Boolean | False    | true         | When it is set to `true`, include topic partition number to the object path.                                                                                                                                     |
+| `bytesFormatTypeSeparator`      | String  | False    | "0x10"       | It is inserted between records for the `formatType` of bytes. By default, it is set to '0x10'. An input record that contains the line separator looks like multiple records in the output object.                |
+| `pendingQueueSize`              | int     | False    | 100          | The number of records buffered in queue, by default it will always be `batchSize * 10`, and it can be set manually.                                                                                              |
+| `useHumanReadableSchemaVersion` | Boolean | False    | false        | Use a human-readable format string for schema version in the message metadata. If it is set to `true`, the schema version is in plain string format. Otherwise, the schema version is in hex-encoded string.     |
```

Comment:
```suggestion
| `useHumanReadableSchemaVersion` | Boolean | False    | false        | Use a human-readable format string for the schema version in the message metadata. If it is set to `true`, the schema version is in plain string format. Otherwise, the schema version is in hex-encoded string format.     |
```